render-test-config.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# TODO: Add Janitor secrets here
+# https://broadworkbench.atlassian.net/browse/ID-96
+# Call from GitHub actions once Janitor is integrated
+
+VAULT_TOKEN=${1:-$(cat $HOME/.vault-token)}
+DSDE_TOOLBOX_DOCKER_IMAGE=broadinstitute/dsde-toolbox:consul-0.20.0
+VAULT_AZURE_MANAGED_APP_CLIENT_PATH=secret/dsde/terra/azure/common/managed-app-publisher
+AZURE_MANAGED_APP_CLIENT_OUTPUT_FILE_PATH="$(dirname $0)"/src/test/resources/azure_managed_app_client.json
+AZURE_PROPERTIES_OUTPUT_FILE_PATH="$(dirname $0)"/src/test/resources/application.conf
+
+docker run --rm --cap-add IPC_LOCK \
+            -e VAULT_TOKEN=$VAULT_TOKEN ${DSDE_TOOLBOX_DOCKER_IMAGE} \
```

Comment:
Nit:
```suggestion
            -e VAULT_TOKEN=$VAULT_TOKEN \
            ${DSDE_TOOLBOX_DOCKER_IMAGE} \
```
It honestly took me several rereadings of the command to find the image argument because my brain didn't expect it to be at the end of the `-e` line.